pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1998,3 +2004,11 @@ type ScrapeClass struct {
 	// +optional
 	AttachMetadata *AttachMetadata `json:"attachMetadata,omitempty"`
 }
+
+// OTLPConfig is the configuration for writing to the OTLP endpoint.
+//
+// +k8s:openapi-gen=true
+type OTLPConfig struct {
+	// +optional
```

Comment:
We could add a description here and also seems reasonable to add a Minimum length validation.
```suggestion
        // List of OTel Attributes that should be promoted to metric labels, defaults to none.
        // +kubebuilder:validation:MinLength=1
	// +optional
```

Reply:
We may also want to prevent duplicate values. WDYT?
```suggestion
        // List of OTel Attributes that should be promoted to metric labels, defaults to none.
        // +kubebuilder:validation:MinLength=1
        // +kubebuilder:validation:listType=map
        //
	// +optional
```

Reply:
> We may also want to prevent duplicate values. WDYT?

Great. Verify duplicate values